rpm/poppler.spec

```diff
@@ -119,42 +120,44 @@ rm -f %{buildroot}%{_libdir}/*.la
 %postun glib -p /sbin/ldconfig
 
 %files qt5
-%defattr(-,root,root,-)
 %{_libdir}/libpoppler-qt5.so.*
 
 %files qt5-devel
-%defattr(-,root,root,-)
 %{_libdir}/libpoppler-qt5.so
 %{_libdir}/pkgconfig/poppler-qt5.pc
-%{_includedir}/poppler/qt5
+%dir %{_includedir}/poppler
```

Comment:
Don't think there's any reason for this? It's already owned by -devel and qt5-devel depends on it.

Reply:
It still has to be owned by the package. It's one of these things that will be errors with RPM 4.20.

See here for example the openSUSE package below if I remove the exact same line in their poppler package:
https://build.opensuse.org/package/live_build_log/home:Thaodan:branches:X11:common:Factory/poppler:qt6/openSUSE_Tumbleweed/x86_64

Reply:
It is already owned by a package. This is a duplicate entry.